demisto_sdk/commands/common/tests/test_files/rn_header_test_data

```diff
@@ -0,0 +1,59 @@
+#### Integrations
+##### integration-test
+- Added x y z
+
+#### Scripts
+##### New: script-test
+- Added x y z
+
+#### Playbooks
+##### playbook-test.yml
+- Added x y z
+- Ignore the yml suffix, it is not a real playbook.
```

Comment:
why does it pass then? (if it should - please add a test case for invalid RN headers)

Reply:
The `TestSuite` creates the content items this way, I just followed